content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -1321,3 +1351,16 @@ See [Seeding your database](/guides/migrate/seed-database) <span class="guide"><
 ```
 
 This is available from version 3.0.1 and later.
+
+## Using a HTTP proxy for the CLI
+
+Prisma CLI supports [custom HTTP proxies](https://github.com/prisma/prisma/issues/506). This is particularly relevant when being behind a corporate firewall.
+
+To activate the proxy, provide the environment variables `HTTP_PROXY` and/or `HTTPS_PROXY`. The behavior is very similar to how the [`npm` CLI handles this](https://docs.npmjs.com/misc/config#https-proxy).
```

Comment:
```suggestion
To activate the usage of a proxy, provide the environment variables `HTTP_PROXY` and/or `HTTPS_PROXY`. The behavior is very similar to how the [`npm` CLI handles this](https://docs.npmjs.com/misc/config#https-proxy).
```

Reply:
Let me know if this section reads better